shell/browser/file_select_helper.cc

```diff
@@ -165,10 +164,7 @@ void FileSelectHelper::OnListFile(
 void FileSelectHelper::LaunchConfirmationDialog(
     const base::FilePath& path,
     std::vector<ui::SelectedFileInfo> selected_files) {
-  ShowFolderUploadConfirmationDialog(
-      path,
-      base::BindOnce(&FileSelectHelper::ConvertToFileChooserFileInfoList, this),
-      std::move(selected_files), web_contents_);
```

Comment:
We were incorrectly relying on the non toolkit view implementation of `ShowFolderUploadConfirmationDialog` before https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/browser_dialogs.cc;l=34-44, this was made obvious when `//ui/views/controls/webview` was included as a dependency which defines `TOOLKIT_VIEWS`.

The change just aligns with what was expected, if upstream dialog implementation is required then the correct dependencies should be included in the future `//chrome/browser/ui/views/folder_upload_confirmation_view.cc`.